qemu/tests/qemu_guest_agent.py

```diff
@@ -102,49 +101,45 @@ def _cleanup_open_session(self):
         except Exception:
             pass
 
-    @error.context_aware
-    def gagent_install(self, params, vm, *args):
+    def gagent_install(self, params, test, vm, *args):
         if args and isinstance(args, tuple):
             gagent_install_cmd = args[0]
         else:
-            raise error.TestError("Missing config 'gagent_install_cmd'")
+            test.error("Missing config 'gagent_install_cmd'")
 
         if not gagent_install_cmd:
             return
 
-        error.context("Try to install 'qemu-guest-agent' package.",
-                      logging.info)
+        logging.info("Try to install 'qemu-guest-agent' package.")
```

Comment:
should use error_context.text()

Reply:
@suqinhuang  Thanks,  No has "text" method in error_context.   I try to use error_context.context() to replace logging.info.  But it hit an error "IndexError: list assignment index out of range".   I haven't find reason.  so continue to use logging.info.  Did you hit this error?

Reply:
@suqinhuang  As remove @error_context.context_aware from file.  so cann't use error_context.